tests/e2e/specs/stories-editor/block-manager-disabled.test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { createNewPost, getAllBlocks, selectBlockByClientId } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { activateExperience, deactivateExperience } from '../../utils';
+
+describe( 'Block Manager', () => {
+	beforeAll( async () => {
+		await activateExperience( 'stories' );
+	} );
+
+	afterAll( async () => {
+		await deactivateExperience( 'stories' );
+	} );
+
+	it( 'the button should not appear, due to styling of display: none', async () => {
+		await createNewPost( { postType: 'amp_story' } );
+
+		// Select the page block.
+		await selectBlockByClientId(
```

Comment:
Why is that needed? The 'More tools & options' button should work without selecting a block.

Reply:
Good question. The test doesn't pass without that, but I'll look for alternatives.

Reply:
`npm run test:e2e:interactive` helps with debugging.

In a manual quick test in the browser I did not need to select a block for this to work.

Perhaps you just need an `await page.waitForSelector(...)` here instead.

Reply:
Actually, there's a nice `clickBlockToolbarButton` utility in Gutenberg that can be used for this.

Example use case: `await clickBlockToolbarButton( 'More options' );`

Reply:
Thanks, `npm run test:e2e:interactive` was really helpful! 

Thanks for bringing up `clickBlockToolbarButton()`. It looks like [it queries inside](https://github.com/WordPress/gutenberg/blob/2f757fa8012edb5cd684a26adef9cb4e8c2158db/packages/e2e-test-utils/src/click-block-toolbar-button.js#L7) `'.block-editor-block-toolbar'`, and didn't seem to find the right button.

https://github.com/ampproject/amp-wp/pull/2937/commits/79e5d057ba6f610fcbfe84aa4cbf17de3730272c removes the `selectBlockByClientId()` call. 

I think the problem here was that the previous `toMatch()` call simply looked for the text of `'Block Manager'` in the DOM, whether it was visible or not.

So this uses [isIntersectingViewport()](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#elementhandleisintersectingviewport). 

It also uses XPath, though I know [you prefer to avoid it](https://github.com/ampproject/amp-wp/issues/2888). This is to target the text of `'Block Manager'` inside the `<button>`. Otherwise, if this just targeted the CSS selector of the `<button>`, the markup might change, and this test wouldn't catch if the text of 'Block Manager' displayed.

Reply:
Ugh, sorry, I meant `clickOnMoreMenuItem`.

Reply:
Oh, all good.